pkg/monitortests/testframework/watchevents/event.go

```diff
@@ -222,8 +222,6 @@ func recordAddOrUpdateEvent(
 		Message(message).Build(pathoFrom, to)
 
 	logrus.WithField("event", *obj).Info("processed event")
-	logrus.WithField("locator", interval.StructuredLocator).Info("resulting interval locator")
```

Comment:
Wanna take out the processed event too? i dont see a reason to keep it at this point, nobodys going to be looking for that.

Reply:
removed and pushed. thx